django_graph_api/graphql/types.py

```diff
@@ -246,6 +247,13 @@ class Character(Object):
     def __init__(self, object_type):
         self.object_type = object_type
 
+    def bind(self, selection, obj):
+        super(RelatedField, self).bind(selection, obj)
+        if self.object_type == 'self':
+            self.object_type = obj.__class__
+        elif is_lambda(self.object_type):
```

Comment:
maybe this should just use [callable](https://docs.python.org/3/library/functions.html#callable) (if this route ends up being kept)?